src/game_engine/unity/mono.rs

```diff
@@ -480,6 +481,162 @@ impl Class {
     }
 }
 
+/// A Mono-specific implementation useful for automatic pointer path resolution
+pub struct Pointer<const N: usize> {
+    static_table: Option<Address>,
+    offsets: [u32; N],
+    is_64_bit: bool,
+}
+
+impl<const N: usize> Pointer<N> {
+    /// Creates a new instance of the Pointer struct
+    pub const fn new() -> Self {
+        Self {
+            static_table: None,
+            offsets: [0; N],
+            is_64_bit: false,
+        }
+    }
+
+    /// Removes the internally stored address of the Mono clas    ///
```

Comment:
Is this comment `Removes` a typo, should it be `Resolves` instead?

Reply:
Yep, apparently formatting the document before sending the PR messed up with some comments.